include/mbedtls/config.h

```diff
@@ -1541,6 +1541,9 @@
  *
  * \note Currently compression can't be used with DTLS.
  *
+ * \deprecated This feature is deprecated and will likely be removed
```

Comment:
Can you please make this more certain? Something like:
```
\deprecated This feature will be removed in a future version of the library.
```

Reply:
@sbutcher-arm This is the wording we've used before (for `MBEDTLS_SSL_TRUNCATED_HMAC_COMPAT`). Why should we use different wording this time?

Reply:
Messages about deprecated features need to be clear and unambiguous, so users know that we intend to remove a feature and they cannot rely on it. This wording offers removal as a possibility, which is not strong enough.

Just because the wording was previously accepted into the library, it doesn't mean its appropriate.